test/data/IdealSolidSolnPhaseTabulatedThermo.cti

```diff
@@ -0,0 +1,31 @@
+#=====================================================================================
+#   Electrodes
+#=====================================================================================
+
+# Graphite (anode)
+#------------------------------------------------------------------
+IdealSolidSolutionTabulatedThermo(
+                                  name = "anode",
```

Comment:
This line, and the following lines, can be indented by only 4 spaces to limit the column width